compiler/rustc_resolve/src/lib.rs

```diff
@@ -1225,10 +1225,7 @@ impl<'tcx> Resolver<'_, 'tcx> {
         );
 
         // FIXME: remove `def_span` body, pass in the right spans here and call `tcx.at().create_def()`
```

Comment:
Is this fixme up to date?

Reply:
Yes, https://github.com/rust-lang/rust/pull/118633 is trying to address it, but there are some complications.